tests/integration/coordinator/plugins/clickhouse/test_plugin.py

```diff
@@ -215,6 +218,13 @@ async def check_object_storage_data(cluster: List[ClickHouseClient]) -> None:
         assert response == expected_data
 
 
+@pytest.mark.asyncio
+async def test_restores_url_engine_table(restored_cluster: List[ClickHouseClient]) -> None:
+    for client in restored_cluster:
+        response = await client.execute(b"SELECT count() FROM system.tables WHERE name = 'url_engine_table'")
```

Comment:
Can you check the full `create_table_query` from system.tables instead of just the name ?

It's to make sure we didn't lose any part of the table definition.

Reply:
Done